src/linear_algebra/dual.lean

```diff
@@ -347,13 +499,25 @@ variables (K V)
 /-- A vector space is linearly equivalent to the dual of its dual space. -/
 def eval_equiv [finite_dimensional K V] : V ≃ₗ[K] dual K (dual K V) :=
 linear_equiv.of_bijective (eval K V)
-  (ker_eq_bot.mp eval_ker) (range_eq_top.mp erange_coe)
+  (by { rw ← ker_eq_bot, apply eval_ker K V }) -- 60x faster elaboration
```

Comment:
Could you add the old version in the comment? Otherwise it won't be super useful :sweat_smile: